src/components/Avatar.js

```diff
@@ -5,5 +5,11 @@
 export default function Avatar(props) {    
     const { person } = props;
 
-    // return ...
+    return (
+        <img
+            classname = "Avatar"
```

Comment:
there is classname attribute in React, it should be written in camelCase styling